### PR TITLE
Fix #2541 - Value field should be mandatory in add collateral

### LINKED ISSUE #2541: Transaction Details in Recurring Deposit account
Go to Client >> New Recurring Deposit Application >> Approve >> Activate 
Deposit >> Transactions >> Details are not well aligned
ree2

I think it suppose to look like when editing/undo that Transaction Details

### PR DESCRIPTION
**Pull request by MYKatz**

### CODE REVIEW

**Comment by mbj36** [comment]
Thanks for the PR, It will be reviewed Shortly!

This message was posted by [GitMate.io](https://gitmate.io) with timestamp signature Thanks for the PR, It will be reviewed Shortly!:1eKruk:AeDrkjaLliveaNPSn5jHZgD0ugw

File: app/views/loans/addloancollateral.html [hunk 1]
```diff
@@ -23,11 +23,17 @@ <h4>{{ 'label.heading.addloancollateral' | translate }}</h4>
...
+                            class="required">*</span></label>
 
                     <div class="col-sm-3">
-                        <input id="value" type="text" class="form-control" ng-model="formData.value">
+                        <input id="value" name="value" type="text" class="form-control" ng-model="formData.value" required late-validate>
                     </div>
+                    
+                    <div class="col-sm-2">
+                 						<form-validate valattributeform="addloancollateralform" valattribute="value"/>
+					</div>
```
Comment (gkrishnan724) [inline app/views/loans/addloancollateral.html]: Indentation please :)

**Comment by gkrishnan724** [comment]
@MYKatz You don't need to make a new pull-request you could just do the changes in the same branch.  After making your changes do the following:

1. Add your changes using `git add`
2. `git commit --amend`.
3. `git push -f origin develop`. 

That should be it. The PR will automatically be updated. ping me if you have any doubts

**Comment by MYKatz** [comment]
@gkrishnan724 I've added the changes you've suggested to the commit.

**Comment by gkrishnan724** [comment]
Thanks @MYKatz :)

File: app/views/loans/addloancollateral.html [hunk 2]
```diff
@@ -23,11 +23,17 @@ <h4>{{ 'label.heading.addloancollateral' | translate }}</h4>
...
 
                     <div class="col-sm-3">
-                        <input id="value" type="text" class="form-control" ng-model="formData.value">
+                        <input id="value" name="value" type="text" class="form-control" ng-model="formData.value" required late-validate>
+                    </div>
+                    
+                    <div class="col-sm-2">
+                        <form-validate valattributeform="addloancollateralform" valattribute="value"/>
                     </div>
+
```
Comment (ankit01ojha) [inline app/views/loans/addloancollateral.html]: @MYKatz could you remove this extra line.

**Pull request merged**